client/main.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+import React from 'react';
+import { render } from 'react-dom';
+import Http from 'axios/dist/axios';
```

Comment:
I agree this was following some unnecessary (or even wrong) conventions.
About the route, `axios` was confusing `Rollup` which was used in the first run and this was the only way around it. But it should've been changed after `webpack` switch! Mea culpa 😄